submissions/xT0PERx/Popup-style/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Задание на tullbar</title>
+
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+    <body>
+        <div class="container">
+            <div class="div1"><p>Mail</p></div>
+            <div class="div2"><p>Images</p></div>
```

Comment:
Add styles on `:hover`, `:focus`, and `:active` state to all interactive elements.